azure-pipelines/dotnet-vscode-csharp-insertion.yml

```diff
@@ -0,0 +1,98 @@
+trigger: none
+pr: none
+
+parameters:
+  - name: targetBranch
+    displayName: Target Branch for PR
+    type: string
+    default: main
+
+resources:
+  repositories:
+  - repository: 1ESPipelineTemplates
+    type: git
+    name: 1ESPipelineTemplates/1ESPipelineTemplates
+    ref: refs/tags/release
+  - repository: RoslynMirror
+    type: git
+    name: internal/dotnet-roslyn
+  pipelines:
+    - pipeline: officialBuildCI
+      source: dotnet-roslyn-official
+      project: internal
+      branch: main
+      trigger: none
+
+variables:
+  - name: RoslynEndSHA
+    value: $(resources.pipeline.officialBuildCI.sourceCommit)
+  - name: RoslynBuildNumber
+    value: $(resources.pipeline.officialBuildCI.runName)
+  - name: RoslynBuildId
+    value: $(resources.pipeline.officialBuildCI.runID)
```

Comment:
Add this variable group here as well, so this pipeline has access to the PAT for the bot below.
```
- group: dotnet-vscode-insertion-variables
```